common/io/Serial.cpp

```diff
@@ -263,9 +262,6 @@ bool AcquireUUCPLockAndOpen(const std::string &path, int oflag, int *fd) {
   // further opens.
   if (ioctl(*fd, TIOCEXCL) == -1) {
     OLA_WARN << "TIOCEXCL " << path << " failed: " << strerror(errno);
-    close(*fd);
-    RemoveUUCPLockFile(lock_file);
```

Comment:
We should definitely try and tidy up shouldn't we so we don't leave a stale file?